### PR TITLE
Quick clarification about unequal groups

### PR DESCRIPTION
**Pull request by BKaneAtWork**

### CODE REVIEW

**Comment by BKaneAtWork** [comment]
@couchbase/docs-team please review

File: modules/learn/pages/clusters-and-availability/groups.adoc [hunk 1]
```diff
@@ -66,9 +66,9 @@ A number of constraints come into play when allocating active and replica vBucke
 
 *replica balance*:: there should be approximately the same number of replica vBuckets on each node in the cluster.
 
-Not all the constraints can be satisfied when the buckets are allocated across uneven groups.
-In this scenario, the `active balance` and `rack-zone` constraints will take priority:
-when the vBucket map is generated, we will ensure that there are approximately the same number of active vBuckets on each cluster and that replicas of a given vBucket must reside in separate groups.
+Not all the constraints can be satisfied when the buckets are allocated across uneven groups. 
+In this scenario, the `active balance` and `rack-zone` constraints will take priority:
```
Comment (simon-dew) [inline modules/learn/pages/clusters-and-availability/groups.adoc]: Spaces at line end?

```suggestion
Not all the constraints can be satisfied when the buckets are allocated across uneven groups.
In this scenario, the `active balance` and `rack-zone` constraints will take priority:
```

**Review by simon-dew — CHANGES_REQUESTED** [review]

**Review by simon-dew — APPROVED** [review]
LGTM 👍

**Comment by simon-dew** [comment]
Thanks for the contribution!

**Pull request merged**